modules/mod_ginger_admin/package.json

```diff
@@ -10,16 +10,18 @@
     "author": "Driebit",
     "license": "UNLICENSED",
     "devDependencies": {
-        "autoprefixer": "^6.7.4",
-        "glob": "^5.0.1",
+        "babel-core": "^6.26.3",
+        "babel-preset-env": "^1.7.0",
+        "breakpoint-sass": "^2.7.1",
         "gulp": "^3.9.1",
-        "gulp-autoprefixer": "^3.1.1",
-        "gulp-css-globbing": "^0.1.9",
-        "gulp-livereload": "^3.8.1",
-        "gulp-plumber": "^1.1.0",
-        "gulp-postcss": "^6.4.0",
-        "gulp-rename": "^1.2.2",
-        "gulp-sass": "^3.1.0",
-        "gulp-uglify": "^3.0.0"
+        "gulp-babel": "^7.0.0",
+        "gulp-css-globbing": "^0.2.2",
+        "gulp-livereload": "^4.0.1",
+        "gulp-plumber": "^1.2.0",
+        "gulp-postcss": "^8.0.0",
+        "gulp-rename": "^1.4.0",
+        "gulp-sass": "^4.0.2",
+        "gulp-uglify": "^3.0.1",
+        "lost": "^8.3.0"
```

Comment:
These changes seem unrelated. Were they intended as a clean-up or update?

Reply:
For the new gulp setup.